test/python/Dockerfile

```diff
@@ -193,6 +193,20 @@ RUN set -ex; \
   cd h-pipenv; \
   pipenv lock;
 
+#--------------------------------------
+# test pdm: pdm (multiple python)
+#--------------------------------------
+FROM build as test-pdm
+
+USER 1000
+
+# make as latest
+# renovate: datasource=github-releases packageName=containerbase/python-prebuild
+RUN install-tool python 3.11.3
```

Comment:
isn't python already on build stage?

Reply:
The tests for pipenv do also install python on each stage. https://github.com/containerbase/base/pull/973/files#diff-8722b87dfd4067380b8581bb8d4727b2bedaa8d620571984262fabc897af38b3R183

Reply:
not all, only some to test older python. so remove to speed up tests